e2eshark/pytorch/models/opt-125M/model.py

```diff
@@ -1,7 +1,10 @@
-import sys, argparse
-import torch
```

Comment:
Oops, this broke this model when tested via turbine.

```
Traceback (most recent call last):
  File "D:\dev\projects\SHARK-TestSuite\e2eshark\test-run\pytorch\models\opt-125M\runmodel.py", line 145, in <module>
    torch.save(E2ESHARK_CHECK["input"], inputsavefilename)
    ^^^^^
NameError: name 'torch' is not defined
```

This stub file was missing its own `import torch`: https://github.com/nod-ai/SHARK-TestSuite/blob/main/e2eshark/tools/stubs/turbinemodel.py

Reply:
Fix: https://github.com/nod-ai/SHARK-TestSuite/pull/245